### PR TITLE
Added 2018 confirmed speakers

### PR DESCRIPTION
**Pull request by ahmedammar841**

### CODE REVIEW

File: source/components/rp2017/speakers/speakers.js [hunk 1]
```diff
@@ -13,6 +13,12 @@ class Speakers extends Component {
 
         const speakerList = [
              // Keynotes
+             {
+                'name': 'Siraj Raval',
+                'img': '../assets/img/keynotes/siraj.jpg',
+                'year': 'YouTuber - Accessible CS education',
+                'wikipedia': 'https://twitter.com/sirajraval?lang=en'
```
Comment (PradyumnaShome) [inline source/components/rp2017/speakers/speakers.js]: @modnikhil Are we OK with non-Wikipedia links?

Comment (itsmesatwik) [reply source/components/rp2017/speakers/speakers.js]: Should be fine as long as we specifically don't say "wikipedia page"

**Comment by itsmesatwik** [comment]
I've merged it into migrate_2017_info, if you're done with the branch could you delete it.

**Comment by ahmedammar841** [comment]
I'll leave the branch open to use for the adjustments to speakers we discussed during this week's meeting.

**Pull request merged**